os/include/tinyara/os_api_test_drv.h

```diff
@@ -103,21 +103,21 @@ extern "C" {
 #endif
 
 /****************************************************************************
- * Name: kernel_test_drv_register
+ * Name: os_api_test_drv_register
  *
  * Description:
- *   This function creates a device node like "/dev/kernel_test" which will be used
- *   by the tests that execute kernel side APIs
+ *   This function creates a device node like "/dev/os_api_test" which will be used
+ *   by the tests that execute os(kernel, network and fs) side APIs
```

Comment:
Let's write OS instead of os because it's description, not name.

Reply:
corrected